### PR TITLE
feat: Adding a new page for STAR alignment

### PR DESCRIPTION
**Pull request by PeterHHer**
Created md file

### CODE REVIEW

File: STAR.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+- Paired-end or single-end sequencing data.
+
+## Index
+
+## 2. Generating a Genome Index
+
+STAR requires a genome index to efficiently align reads. Submit the following command in your SLURM script to generate it:
+
+```bash
+module load STAR/version_number
```
Comment (JoshuaSiraj) [inline STAR.md:55]: Make the use of version_number as a variable more explicit
```suggestion
module load STAR/<version_number>
# Replace <version_number> with the desired version, e.g., module load STAR/2.7.9a
```

File: STAR.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+- **`--sjdbOverhang`**: Read length minus 1 (e.g., for 101 bp reads, use 100).
+
+## 3. Aligning RNA-seq Reads
+
+After generating the genome index, you can align RNA-seq reads using STAR.
+
+### Single-End Reads Example:
+
+```bash
+module load STAR/version_number
```
Comment (JoshuaSiraj) [inline STAR.md:80]: Make the use of version_number as a variable more explicit

```suggestion
module load STAR/<version_number>
# Replace <version_number> with the desired version, e.g., module load STAR/2.7.9a
```

File: STAR.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+     --outFileNamePrefix sample \
+     --quantMode GeneCounts \
+     --outReadsUnmapped Fastx \
+     --chimSegmentMin 10 \
+     --outSAMtype BAM SortedByCoordinate
+```
+### Paired-End Reads Example:
+
+```bash
+module load STAR/version_number
```
Comment (JoshuaSiraj) [inline STAR.md:95]: Make the use of version_number as a variable more explicit

```suggestion
# Replace <version_number> with the desired version, e.g., module load STAR/2.7.9a
```

File: STAR.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+## Running STAR Alignments in Parallel Using `sbatch --array`
+
+When processing multiple RNA-seq samples, you can use SLURM's `--array` option to run jobs in parallel. This approach is efficient for handling batch alignment tasks.
+
+### 1. Preparing Input Files
+
+Create a text file listing all your sample FASTQ file pairs, with one sample per line. For paired-end data, include both files separated by a space, like this:
+
+**`samples.txt`:**
```
Comment (JoshuaSiraj) [inline STAR.md:193]: Assuming there is supposed to be a file here?

File: docs/disciplines/Bioinformatics/STAR.md
Comment (JoshuaSiraj) [inline docs/disciplines/Bioinformatics/STAR.md:1]: Move content from the other STAR.md to here

File: STAR.md
Comment (JoshuaSiraj) [inline STAR.md:1]: Move content to docs/disciplines/Bioinformatics/STAR.md and delete

**Review by JoshuaSiraj — CHANGES_REQUESTED** [review]
Good work! Just need some changes

**Comment by jjjermiah** [comment]
@PeterHHer can you please review these changes

**Pull request merged**